utility/src/main/java/org/oppia/util/firebase/CrashLogger.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.util.firebase
```

Comment:
Suggest moving this to a non-Firebase specific project since the interface isn't part of the implementation. Maybe org.oppia.util.logging?